mmv1/products/backupdr/BackupVault.yaml

```diff
@@ -87,6 +89,29 @@ parameters:
     min_version: 'beta'
     url_param_only: true
     default_value: false
+    deprecation_message: '`force_delete` is deprecated and will be removed in a future major release. Use ignore_inactive_datasources` instead.'
+    exactly_one_of:
```

Comment:
`exactly_one_of` means one of the two fields is required, but `force_delete` is optional before. It is a breaking change to the exiting resource without `force_delete` in the configuration.

Instead, `conflicts` can be used.